tests/Agent/IntegrationTests/IntegrationTests/RemoteServiceFixtures/WebApiAsyncFixture.cs

```diff
@@ -85,13 +87,30 @@ public void GetSync_Sync()
             DownloadJsonAndAssertEqual(address, "Worked");
         }
 
-
         public void ExecuteResponseTimeTestOperation(int delayDurationSeconds)
         {
             var address = $"http://localhost:{Port}/ResponseTime/CallsOtherMethod/{delayDurationSeconds}";
             DownloadJsonAndAssertEqual(address, "Worked");
         }
 
+        public string Request(HttpMethod method)
+        {
+            using (var httpClient = new HttpClient())
+            {
+                var requestMessage = new HttpRequestMessage(method, $"http://localhost:{Port}/AsyncFireAndForget/Sync_Sync");
+                var result = httpClient.SendAsync(requestMessage).Result;
+
+                return result.Content.ReadAsStringAsync().Result;
+            }
+        }
 
+        public void Get404(string Path = "DoesNotExist")
+        {
+            try
+            {
+                new WebClient().DownloadString($"http://localhost:{Port}/{Path}");
+            }
+            catch (WebException) { }
```

Comment:
Should we emit this error for the tests? It might cause some false positives?

Reply:
An expected 404 error is thrown here that we want swallowed.